src/Shrink.elm

```diff
@@ -298,6 +298,7 @@ tuple4 (shrinkA, shrinkB, shrinkC, shrinkD) (a, b, c, d) =
   +++ Lazy.List.map2 (\a b   -> (a,b,c,d)) (shrinkA a) (shrinkB b)
   +++ Lazy.List.map3 (\b c d -> (a,b,c,d)) (shrinkB b) (shrinkC c) (shrinkD d)
   +++ Lazy.List.map3 (\a c d -> (a,b,c,d)) (shrinkA a) (shrinkC c) (shrinkD d)
+  +++ Lazy.Lest.map3 (\a b d -> (a,b,c,d)) (shrinkA a) (shrinkB b) (shrinkD d)
```

Comment:
I did, but it looks like I did the "run the compiler in the wrong directory" trick :(

Anyway, I can push a fix.